### ISSUE TITLE
Access to hardwares like webcam via the wsl

### ISSUE THREAD

**Issue by banzade** [microsoft/WSL#5218]
**Is your feature request related to a problem? Please describe.**
I am using OpenCV for my project. I recently started using wsl for all my python devs, but unfortunately I noticed that WSL can't access the device hardwares like the webcam; which is really unfortunate considering how cool wsl is. 
**Describe the solution you'd like**
Well WSL should atleast incorporate the webcam for commands like `cv2.VideoCapture(-1)` for OpenCV

**Comment by Rajarshi07** [comment]
Have you found any solution to this issue? (other than vmware or dualbooting ubuntu)

**Comment by banzade** [comment]
Not any as of yet. It seems that even WSL2 can't access the integrated webcam. Well for OpenCV and accessing the webcam, I installed OpenCV in Windows. There was no problem whatsoever.

**Comment by Biswa96** [comment]
If the webcam is on any network it may be possible to access it in WSL2 VM also.

**Comment by banzade** [comment]
Yes if you stream via your webcam. But I dont see the point of going through all that, when it works in windows 10 itself. What I really want is for WSL2 to have full access to the hardware not just the GPU but the webcam too.
